helm/nostr/strfry-chart/templates/configmap.yml

```diff
@@ -0,0 +1,145 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "strfry-chart.fullname" . }}-configmap
```

Comment:
imo "-configmap" is redundant. Unless there's a reason for keeping it, I'd just get rid of it

Reply:
idk, all other templates do have a suffix, it would seem odd for this one not to have it? also other  templates of charts in the repo have suffixes as well, so it looks like a convention?